src/main/java/io/stargate/sgv2/jsonapi/api/model/command/deserializers/SortClauseDeserializer.java

```diff
@@ -51,36 +52,69 @@ public SortClause deserialize(JsonParser parser, DeserializationContext ctxt) th
     while (fieldIter.hasNext()) {
       Map.Entry<String, JsonNode> inner = fieldIter.next();
       String path = inner.getKey().trim();
-
+      float[] arrayVals = null;
```

Comment:
Maybe rename to `vectorFloats` -- `array` here might seem to relate JSON Number Array value (the other Vector representation).